demo/views/demo-all.dust

```diff
@@ -34,14 +35,11 @@
   </div>
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
Sorry, missed the last reply - We’d have two choices here:

1. Revert the change here for now
2. Introduce working version of CF manifest and fix things accordingly